odfuzz/config.py

```diff
@@ -45,7 +84,10 @@ def async_requests_num(self):
 
 
 class Config:
+    fuzzer = None
+    dispatcher = None
+
     @staticmethod
     def init():
         Config.fuzzer = FuzzerConfig()
-        Config.dispatcher = DispatcherConfig()
\ No newline at end of file
+        Config.dispatcher = DispatcherConfig()
```

Comment:
is this just different newline? imho should not be present in this PR. Leave pls re-formatting away

Reply:
ok.